dist/src/main/java/io/camunda/zeebe/shared/ActorClockConfiguration.java

```diff
@@ -13,7 +13,7 @@
 import io.camunda.zeebe.shared.management.ControlledActorClockService;
 import java.util.Optional;
 import org.springframework.boot.context.properties.ConfigurationProperties;
-import org.springframework.boot.context.properties.ConstructorBinding;
+import org.springframework.boot.context.properties.bind.ConstructorBinding;
```

Comment:
Was already deprecated in 3.0 and now removed see
https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.2-Release-Notes#deprecations-from-spring-boot-30